src/components/common/UserProfileEdit/UserProfileEdit.tsx

```diff
@@ -44,10 +44,10 @@ const UserProfileEdit = () => {
     >
       <Tabs>
         <TabList>
-          <Tab>
+          <Tab tabIndex={0}>
```

Comment:
I was noticing the `focus-visible` outline was occasionally being triggered on click, which I don't think is intended (normally reversed for tabbing). Anecdotally, this has resolved it for me.